<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsciiReaderSettings</class>
 <widget class="QWidget" name="AsciiReaderSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Number Of Channels:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spNumOfChannels">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select number of channels or set to 0 for Auto (determined from incoming data)</string>
     </property>
     <property name="specialValueText">
      <string>Auto</string>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Column Delimiter:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rbComma">
       <property name="text">
        <string>comma</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbSpace">
       <property name="text">
        <string>space</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbTab">
       <property name="text">
        <string>tab</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbOtherDelimiter">
       <property name="text">
        <string>other:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDelimiter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Enter a custom delimiter character</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>|</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Filter by Prefix:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="rbFilterDisabled">
       <property name="text">
        <string>Disabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbFilterInclude">
       <property name="toolTip">
        <string>Only read lines that start with prefix</string>
       </property>
       <property name="text">
        <string>Include</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbFilterExclude">
       <property name="toolTip">
        <string>Do not read lines that start with prefix</string>
       </property>
       <property name="text">
        <string>Exclude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFilterPrefix">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Prefix text used to filter incoming data</string>
       </property>
       <property name="placeholderText">
        <string>prefix text</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Value format:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="cbHex">
     <property name="toolTip">
      <string>Expect HEX data instead of decimal.</string>
     </property>
     <property name="text">
      <string>Hex data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
